Clarify that Now Desktop is not being improved (#698)
zeit/now-desktop: README.md

# Now Desktop

> ---

Reviewer: ```suggestion
```
Looks odd with the header underline also.

> **IMPORTANT:** Now Desktop will not be improved further (with the exception of receiving security updates).

Reviewer: ```suggestion
**IMPORTANT:** Now Desktop is no longer under active development (with the exception of receiving security updates).
```

> In turn, we recommend using [our Git Integration](https://vercel.com/import) instead or installing our [command-line interface](https://vercel.com/download) if you'd like to deploy from your terminal.

Reviewer: ```suggestion
We recommend using [our Git Integration](https://vercel.com/import) instead, or installing our [command-line interface](https://vercel.com/download) if you'd like to deploy from your terminal.
```
